src/main/java/dukeTasksPackage/Task.java

```diff
@@ -0,0 +1,28 @@
+package dukeTasksPackage;
+
+public abstract class Task {
+    public String description;
+    protected int number;
+    public  char status = ' ';
+    public char alphabet = ' ';
+    public boolean isDone;
+    public void markAsDone(Task t) {
+        System.out.println("Good job! You have completed another task! I've marked this task as done:\n" + "    [X] " + t.description);
```

Comment:
You may want to consider wrapping this line around, especially if it exceeds 120 characters, as per the coding standard.